catalog.html

```diff
@@ -214,27 +227,19 @@ <h2>Пара слов о перфораторах</h2>
      </ul>
      <p>
       Обратная связь:<br>
-      <a href="#">mail@htmlacademy.ru</a>
+      <a href="mailto:mail@htmlacademy.ru">mail@htmlacademy.ru</a>
      </p>
      <p>
       Разработано -<br>
       <a href="https://htmlacademy.ru/intensive/htmlcss">htmlacademy.ru</a>
      </p>
    </footer>
-   <form action="https://echo.htmlacademy.ru" method="post">
-    <fieldset>
-      <legend>Форма по наведению курсора на товар</legend>
-      <button type="submit">Купить</button>
-      <button type="submit">В закладки</button>
-    </fieldset>
-   </form>
-   <form>
-     <fieldset>
-       <legend>Форма поп-ап при добавлении товара в корзину</legend>
+   <!--это окно будет отображаться при наведении на товар-->
+   <a href="#"><img src="img/basket.svg" alt="Иконка корзины для покупок">Купить</a>  <!--не оторабажается иконка-->
+   <a href="#">В закладки</a>  
```

Comment:
Это не модальное окно, это всплывающий блок и он должен быть размечен в каждой карточке товара, а не один на все. Для блока нужна общая обёртка:
```suggestion
<div>
   <a href="#">Купить</a>
   <a href="#">В закладки</a>
</div>
```
Иконка - декоративный элемент.